main.py

```diff
@@ -28,6 +29,10 @@ def getallus(chatroom):
     return userlist
 
 
+def ipaddress():
+    return get('https://api64.ipify.org').text
```

Comment:
This does a great job in retrieving an IPv6 address as evidenced in the screenshot I have attached just below.

![image](https://user-images.githubusercontent.com/49605954/96952289-1b7e8780-150c-11eb-826b-632d6ff6d64d.png)

But when it comes to IPv4, it still ends up fetching the IPv6 address.

![image](https://user-images.githubusercontent.com/49605954/96952331-34873880-150c-11eb-89e6-b05bf6b8635a.png)